app/views.py

```diff
@@ -0,0 +1,9 @@
+from app import app
```

Comment:
VIews should be a folder with a list of files for the various views. For example ./vIew/user.py  where user.py contains user views.

Reply:
Sure I will add them